2021/_layouts/default.html

```diff
@@ -27,8 +27,18 @@
     <meta property="og:title" content="{{site.data.homepage.og_title}}"/>
     <meta property="og:description" content="{{site.data.homepage.og_description}}"/>
     <meta property="og:type" content="website"/>
-    <meta property="og:image" content="{{ base }}{{site.data.homepage.og_image}}"/>
+    <meta property="og:image" content="{{ site.data.homepage.og_url }}{{site.data.homepage.og_image}}"/>
```

Comment:
`{{ base }}` (is in `base.html` I think) is not need anymore?

Reply:
In this case, `{{base}}` is equivalent to. `.` and the crawlers aren't able to locate the image. I changed it to an absolute path and now it's working. I don't know where else it is used.